ppr-ui/tests/unit/FolioNumberSummary.spec.ts

```diff
@@ -56,7 +56,11 @@ describe('Folio number on the summarty page', () => {
 
   it('sets the validity to false for > 15 characters', async () => {
     wrapper.find('#txt-folio').setValue('MY TEST THAT IS VERY LONG IN FACT TOO LONG')
-    expect(wrapper.emitted().folioValid).toBeTruthy()
+    await Vue.nextTick()
+    await Vue.nextTick()
+    await Vue.nextTick()
     expect(wrapper.vm.$data.isValid).toBeFalsy()
+    expect(wrapper.emitted().folioValid).toBeTruthy()
```

Comment:
shouldn't this emit false? Try await flushPromises and use the getLastEvent fn from utils (it will get the last emitted event -- this will have emitted true first and then false after)

Reply:
So, the fact that it emitted an event called folio valid is true. does that make sense? It's not the value of the emit, but that fact that an event was emitted

Reply:
oh that's right! Sorry, this is good then. Probably better to use that util fn in the future though since here we don't know if its emitting the right value or not